Lab1.ipynb

```diff
@@ -0,0 +1,482 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.rcParams[\"figure.figsize\"] = (15,10)\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "\n",
+    "x = np.linspace(0,10,1000);\n",
+    "prob = expon.pdf(x) \n",
+    "#ax.plot(x,prob,'r-',lw=5,alpha=0.6, label='expon pdf')\n",
+    "ax.plot(x, prob,'r-', lw=5, alpha=0.6, label='expon pdf')\n",
+    "\n",
+    "r = expon.rvs(size=100000)\n",
+    "ax.hist(r, density = True)\n",
```

Comment:
I think your plot will look nicer if you can change the number of bins for your histogram. You can set the number of bins with: ` .hist(r, bins = theNumberofBinsYouWant, density = True) `

Reply:
I was wondering how to increase the bar numbers. Thanks for letting me know!